glib/src/value.rs

```diff
@@ -395,6 +432,17 @@ impl Value {
         }
     }
 
+    /// Tries to get a value of an owned type `T`.
+    pub fn get_owned<T>(&self) -> Result<T, <<T as FromValue>::Checker as ValueTypeChecker>::Error>
+    where
+        T: FromOwnedValue,
```

Comment:
```suggestion
        T: for<'b> FromValue<'b> + 'static,
```

Does this maybe also work?

Reply:
If that works we won't need a new trait, if it doesn't then that's just how it is. With the blanket implementation it should be good enough.